test/linalg/dense.jl

```diff
@@ -553,7 +553,7 @@ end
             @test coth(acoth(coth(A))) ≈ coth(A)
 
             # Definition of principal values (Aprahamian & Higham, 2016, pp. 4-5)
-            abstol = 1e-8 * norm(acosh(A))
+            abstol = max(1e-8, eps(real(elty))) * norm(acosh(A))
```

Comment:
Probably `sqrt(eps(real(elty))) * vecnorm(acosh(A))` would do the trick here.  (sqrt(ε) is the default reltol, and we normally do approximate comparisons with `vecnorm` for performance.)

Reply:
OK, that works too.